doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,391 @@
+
+-------------------
+Generate Mesh Files
+-------------------
+
+To generate the ``cylinder.msh`` and ``support.msh`` files, we used the ``gmsh.py`` file. The two functions take as input the bottom's position (x0), the radius, the number of points, and the height. 
+
+.. code-block:: text
+
+  cylinder(x0, radius, n_points, height)
+  support(x0, radius, n_points, height)
+
+
+In this case, the cylinder is 0.02 tall with a radius of 0.005 and the support is 0.005 tall and has the same diameter as the cylinder.
+
+.. code-block:: text
+
+  cylinder(-0.005, 0.005, 50, 0.02)
+  support(-0.01 ,0.005 ,50, 0.005)
+
+.. figure:: images/meshfiles.png
+    :width: 400
+    :alt: Mesh
+    :align: center
+
+Mesh files can be generated using the following command.
+
+.. code-block:: text
+  :class: copy-button
+
+  python3 gmsh.py
+
+--------------
+Parameter File
+--------------
+
+Mesh
+~~~~~
+
+The domain we simulate is a rectangular box measuring :math:`0.045\times0.02\times0.02` meters, made using the deal.ii grid generator.  The grid is refined 6 times using the ``set initial refinement`` parameter.
+
+.. code-block:: text
+
+  subsection mesh
+  set type               = dealii
+  set grid type          = subdivided_hyper_rectangle
+  set grid arguments     = 3,1,1 : -0.010,-0.01,-0.01 : 0.035, 0.01,0.01 : true
+  set initial refinement = 6
+  end
+  
+  
+Boundaries Conditions
+~~~~~~~~~~~~~~~~~~~~~
+
+The bottom of the domain is set as an outlet to allow the particles to leave the simulation domain, thus reducing the computational time of the simulation.
+
+.. code-block:: text
+    
+  subsection DEM boundary conditions
+  set number of boundary conditions = 1
+
+  subsection boundary condition 0
+  set boundary id         = 0
+  set type                = outlet
+  end
+  end
+
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The particles have a polydisperse size distribution with a density of 1922 :math:`kg/m^3`. 400 000 particles would be needed to match the 2 g of sand used in the experiment. However, to reduce the simulation's duration of this example, we insert only 200 000 particles.
+
+According to Hu *et al.* [#hu2022]_ the rolling friction and the surface energy parameters impact the final simulated angle of repose the most. Thus, we simulate twelve combinations of those parameters. The ``rolling friction`` is between 0.3 and 0.7, and the ``surface energy`` is between 0.0010 and 0.0100. The ``granuheap.prm`` file of this example is specifically for a ``rolling friction`` of 0.5 and a ``surface energy`` of 0.0100. 
+
+The water volume fraction of the wet sand used in the experiment is 0.325 %. According to the work of Liefferink *et al.* [#Liefferink2018]_, this water volume fraction matches an ``friction coefficient`` of 0.5. 
```

Comment:
It's good that you have find a reference for the rolling friction coefficient. 

Maybe you could add something saying that this value can be used as a rough estimate of a realistic rolling friction coefficient for this example.